.gitignore

```diff
@@ -124,3 +124,6 @@ public/
 
 # NumPy zip archives created by the examples.
 *.npz
+
+# macOS
+.DS_Store
```

Comment:
I'm willing to check this in, but some people (i.e. those wanting to maintain a minimal project-wide .gitignore) will scream at you for this and tell you to put it into your machine's global .gitignore file.

Example: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
I can do that instead and remove the `.DS_store` here. I did not know about global `.gitignore` before.